unity-renderer/Assets/UIComponents/Scripts/Components/Notification/ChatNotificationMessageComponentView.cs

```diff
@@ -0,0 +1,156 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public interface IChatNotificationMessageComponentView
+{
+    /// <summary>
+    /// Event that will be triggered when the notification is clicked.
+    /// </summary>
+    Button.ButtonClickedEvent onClick { get; }
```

Comment:
replace with `Action`.. it is better to not couple the events with specific ui components because its easier to support multiple implementations or mocking